### PR TITLE
Merge plugin/0.2.1 code into master

### PR DESCRIPTION
**Pull request by guotuofeng**
merge 0.2.1 bug fixes into master

### CODE REVIEW

File: tb_plugin/docs/gpu_utilization.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
                       This overall number is the sum of all kernels' SM_Eff_K weighted by kernel's execution duration, divided by “all steps time”. 
                       It shows GPU Stream Multiprocessors’ utilization. 
                       Although it is finer grained than above “GPU Utilization”, it still can’t tell the whole story. 
                       For example, a kernel with only one thread per block can’t fully utilize each SM. 
 
-* Est. Achieved Occupancy: The bigger, the better. The definition of occupancy is [here](https://docs.nvidia.com/gameworks/content/developertools/desktop/analysis/report/cudaexperiments/kernellevel/achievedoccupancy.htm). 
+* Est. Achieved Occupancy: For most cases such as memory bandwidth bounded kernels, the higher the better. [Reference](http://developer.download.nvidia.com/GTC/PDF/GTC2012/PresentationPDF/S0514-GTC2012-GPU-Performance-Analysis.pdf). The definition of occupancy is [here](https://docs.nvidia.com/gameworks/content/developertools/desktop/analysis/report/cudaexperiments/kernellevel/achievedoccupancy.htm).
```
Comment (gdankel) [inline tb_plugin/docs/gpu_utilization.md:12]: The improvements in performance does not normally scale linearly after some point, but not sure how to best express this in a clear way.

File: tb_plugin/examples/resnet50_profiler_api.py [hunk 2]
```diff
@@ -31,12 +31,12 @@
...
-        wait=2,
-        warmup=3,
-        active=6),
+        wait=1,
+        warmup=1,
+        active=2),
     on_trace_ready=torch.profiler.tensorboard_trace_handler('./result', worker_name='worker0'),
     record_shapes=True,
-    profile_memory=True,
+    profile_memory=True,  # This will take 1 to 2 minutes. Setting it to False could greatly speedup.
```
Comment (gdankel) [inline tb_plugin/examples/resnet50_profiler_api.py:39]: I suggest we also add a similar comment to with_stack

**Review by gdankel — APPROVED** [review]

File: tb_plugin/docs/gpu_utilization.md (hunk 1 above)
Comment (guotuofeng) [reply tb_plugin/docs/gpu_utilization.md:12]: we will consider more clear description in next release.

File: tb_plugin/examples/resnet50_profiler_api.py (hunk 2 above)
Comment (guotuofeng) [reply tb_plugin/examples/resnet50_profiler_api.py:39]: will do in next release.

File: tb_plugin/docs/gpu_utilization.md (hunk 1 above)
Comment (gdankel) [reply tb_plugin/docs/gpu_utilization.md:12]: I had to fix a couple of linter warnings so I changed this as well to "a higher value often translates to better performance, especially when the starting value is very low." Sounds OK?

Comment (guotuofeng) [reply tb_plugin/docs/gpu_utilization.md:12]: Yes, it is ok

**Pull request closed without merging**